ml-data-hub-plugin/src/test/groovy/com/marklogic/gradle/task/CleanJobsDataTaskTest.groovy

```diff
@@ -0,0 +1,53 @@
+package com.marklogic.gradle.task
```

Comment:
This should all be done in JUnit in the ./marklogic-data-hub project as well, where there's better support for writing and running tests too. There's an internal Wiki page with a guide on writing DHF tests.